src/rwlock.rs

```diff
@@ -1,12 +1,15 @@
 use super::shared::{SpinWait, Waiter};
-#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+#[cfg(all(
+    not(usync_tsan_enabled),
+    any(target_arch = "x86", target_arch = "x86_64"),
+))]
 use lock_api::RawRwLock as _RawRwLock;
```

Comment:
Could this use statement be moved into `try_lock_exclusive_assuming` for the x86 specialization?